src/olympia/files/tasks.py

```diff
@@ -104,3 +105,17 @@ def __exit__(*x):
                     except WebextPermissionDescription.DoesNotExist:
                         log.warning('No "%s" permission found to update with '
                                     '[%s] locale' % (perm, locale))
+
+
+@task
+def extract_file_obj_to_git(file_id, channel):
```

Comment:
Do you need this to be a task for future developments ?

Reply:
Correct, in the future we'll call the extraction on upload but asynchronously in the background.